extension/goext/environment.go

```diff
@@ -193,3 +233,35 @@ func MakeNullFloat(value float64) NullFloat {
 		Valid: true,
 	}
 }
+
+// MakeUndefinedString allocates a new undefined string and sets its value
+func MakeUndefinedString(value string) *NullString {
```

Comment:
naming: "MakeUndefinedX" suggests you're making a value with an undefined value - but the code says that the value is valid.
I suggest making MakeNullX return a pointer and remove the MakeUndefinedX family

Reply:
done